src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -0,0 +1,41 @@
+package edu.kis.vh.nursery;
+
+public class DefaultCountingOutRhymer {
+
+    public static final int CAPACITY = 12;
+    public static final int EMPTY = -1;
+    public static final int DEFAULT_VALUE = -1;
+    final private int[] numbers = new int[CAPACITY];
+
+    public int getTotal() {
+        return total;
+    }
+
+    private int total = EMPTY;
+
+    protected void countIn(int in) {
+        if (!isFull())
+            numbers[++total] = in;
+    }
+
+    protected boolean callCheck() {
+        return total == EMPTY;
+    }
+
+    protected boolean isFull() {
+        return total == CAPACITY - 1;
+    }
+
+    protected int peekaTop() {
+        if (callCheck())
+            return DEFAULT_VALUE;
+        return numbers[total];
+    }
+
+    protected int countOut() {
```

Comment:
Visibility modifiers should be changed only for class fields. Classes without any visible methods and fields are useless.